DQM/GEM/plugins/GEMDigiSource.cc

```diff
@@ -208,6 +217,10 @@ void GEMDigiSource::analyze(edm::Event const& event, edm::EventSetup const& even
         bTagVFAT[nIdxVFAT] = true;
       }
     }
+    auto* histChDigi = mapDigiOccPerCh_.FindHist(key4Ch);
+    if (histChDigi != nullptr) {
+      histChDigi->setBinContent(0, 0, mapBitDigiOccOn_[key4Ch].count());
```

Comment:
I don't think this will work as intended.

If you store the number of inactive channels in the bin of a histogram, when multiple histograms are merged the value will not be correct because the number of channels will be summed over.

Reply:
Hi @fwyzard, 

Yep, it should be fixed properly... I've modified that line better (I guess). Could you have a look?